lib/modern_searchlogic/scope_procedure.rb

```diff
@@ -2,7 +2,17 @@ module ModernSearchlogic
   module ScopeProcedure
     def self.included(base)
       base.singleton_class.class_eval do
-        alias_method :scope_procedure, :scope
+        def scope_procedure(name, options = nil)
+          self._defined_scopes << name.to_sym
```

Comment:
minor but we should probably do this _after_ the method is defined?

Reply:
Ah, yeah, good call.

Moved in 3ad6d5554b583111f1baa6f9174c51ced8df8fed